eng/Version.Details.xml

```diff
@@ -11,5 +11,9 @@
       <Uri>https://github.com/dotnet/roslyn</Uri>
       <Sha>c34bb049f1f74256f109794509cd9f0f1c23faea</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.Helix.Sdk" Version="2.0.0-beta.21063.4">
```

Comment:
note that this is the version given by just grabbing from the same DARC channel that we get the Arcade SDK from.